config/config.json

```diff
@@ -153,6 +153,9 @@
         "AboutLink": "https://about.mattermost.com/default-about/",
         "HelpLink": "https://about.mattermost.com/default-help/",
         "ReportAProblemLink": "https://about.mattermost.com/default-report-a-problem/",
+        "AdministratorsGuideLink": "https://about.mattermost.com/administrators-guide/",
```

Comment:
@R-Wang97 if these are blank, do the options disappear from the System Console main menu?